src/xdp/trn_transit_xdp.c

```diff
@@ -525,9 +524,9 @@ int _transit(struct xdp_md *ctx)
 
 	/* The agent may tail-call this program, override XDP_TX to
 	 * redirect to egress instead */
-	if (action == XDP_TX)
+/*	if (action == XDP_TX)
```

Comment:
This change is left intentionally for next changes after some further cleaning up